.github/workflows/main.yml

```diff
@@ -62,116 +62,128 @@ jobs:
+
+      - name: Prepare
+        run: |
+          New-Item -ItemType Directory "${env:GITHUB_WORKSPACE}/binaries/"
+          & "${env:GITHUB_WORKSPACE}/.github/workflows/.craft.ps1" -c --set forceAsserts=true owncloud/owncloud-client
+          & "${env:GITHUB_WORKSPACE}/.github/workflows/.craft.ps1" -c --set srcDir="${env:GITHUB_WORKSPACE}" owncloud/owncloud-client
+          if ($IsWindows) {
+              & "${env:GITHUB_WORKSPACE}/.github/workflows/.craft.ps1" -c dev-utils/nsis
+          } elseif($IsLinux) {
+              & "${env:GITHUB_WORKSPACE}/.github/workflows/.craft.ps1" -c dev-utils/linuxdeploy
+          }
+
+      - name: Install dependencies
+        run: |
+          if ("${{ matrix.target }}" -eq "linux-64-gcc" ) {
+              # build with appimage updater and others
+              & "${env:GITHUB_WORKSPACE}/.github/workflows/.craft.ps1" -c --set enableAppImageUpdater=true owncloud-client
+              & "${env:GITHUB_WORKSPACE}/.github/workflows/.craft.ps1" -c --set enableCrashReporter=true owncloud/owncloud-client
+              & "${env:GITHUB_WORKSPACE}/.github/workflows/.craft.ps1" -c --set enableAutoUpdater=true owncloud/owncloud-client
+          }
+          & "${env:GITHUB_WORKSPACE}/.github/workflows/.craft.ps1" -c --install-deps owncloud/owncloud-client
+
+      - name: QML format lint
+        if: matrix.name  == 'linux-gcc-x86_64'
+        run: |
+          & "${env:GITHUB_WORKSPACE}/.github/workflows/.craft.ps1" -c libs/qt6/qttools
+          & "${env:GITHUB_WORKSPACE}/.github/workflows/.craft.ps1" -c --run pwsh -c "git ls-files *.qml | %{ qmlformat -i `$_}"
+          $diff = git diff
+          if ($diff) {
+            $diff
+            exit 1
+          }
+
+      - name: clang-format format lint
+        if: matrix.name  == 'linux-gcc-x86_64'
+        run: |
+          & "${env:GITHUB_WORKSPACE}/.github/workflows/.craft.ps1" -c libs/llvm
+          & "${env:GITHUB_WORKSPACE}/.github/workflows/.craft.ps1" -c --run git clang-format --force  ${{ github.event.pull_request.base.sha }}..${{ github.event.pull_request.head.sha }}
+          $diff = git diff
+          if ($diff) {
+            $diff
+            exit 1
+          }
+
+      - name: Build
+        if: ${{ !matrix.useSonarCloud }}
+        run: |
+          if ("${{ matrix.target }}" -eq "macos-64-clang-debug" ) {
+              # https://api.kde.org/ecm/module/ECMEnableSanitizers.html
+              # address;leak;undefined
+              # clang: error: unsupported option '-fsanitize=leak' for target 'x86_64-apple-darwin21.6.0'
+              & "${env:GITHUB_WORKSPACE}/.github/workflows/.craft.ps1" -c --set args="-DECM_ENABLE_SANITIZERS='address;undefined'" owncloud/owncloud-client
+          }
+          if ("${{ matrix.target }}" -eq "linux-64-gcc" ) {
+              # build with appimage updater and others
+              & "${env:GITHUB_WORKSPACE}/.github/workflows/.craft.ps1" -c --set enableAppImageUpdater=true owncloud-client
```

Comment:
Do you need to do this again? This is already done in the `Install dependencies` step.

Reply:
it has to be done again to have the env properly setup - at least this is the way I understood this.